tutorials/text/text_intro.py

```diff
@@ -1,6 +1,7 @@
 """
-Text introduction
-=================
+========================
+Text in Matplotlib Plots
+========================
```

Comment:
remove subtitle just below, or merge with title

Reply:
replace matplotlib by Matplotlib when in text, throughout.

Reply:
The first line shows up in a hover-over in the docs, so I think thats why the "subtitle" is there.